modules/oc-mirror-troubleshooting-v2.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oc-mirror-troubleshooting-v2_{context}"]
+== Troubleshooting oc-mirror plugin v2 Image Mirroring Errors
```

Comment:
Can we say `Image Mirroring Errors` does not sound right here, it means that this section is only for troubleshooting such errors but did we not intend to say this section is for `TroubleShooting Errors` ?